### PR TITLE
22 manage levels

### PR DESCRIPTION
**Pull request by nicolasGibaud7**

### CODE REVIEW

File: Classic_RPG/src/entities/Player.java [hunk 1]
```diff
@@ -23,6 +32,37 @@ public void setDefensePosition(boolean defense_position_) {
 		this.defense_position = defense_position_;
 	}
 	
+	public void levelUp() {
+		this.level += 1;
+		this.characteristics.setMax_hp((int)Math.ceil(this.characteristics.getMax_hp() * 1.1));
+		this.setHp(getHp() + (int)Math.ceil(this.characteristics.getMax_hp() * 0.1));
+		this.characteristics.setMax_stamina((int)Math.ceil(this.characteristics.getMax_stamina() * 1.1));
+		this.setStamina(getStamina() + (int)Math.ceil(this.characteristics.getMax_stamina() * 0.1));
+		System.out.println("Player lvl : "  + level + " - max hp : " + characteristics.getMax_hp() + " - max stamina " + characteristics.getMax_stamina());
```
Comment (LavergneC) [inline Classic_RPG/src/entities/Player.java:45]: Show this to the user ?

File: Classic_RPG/src/entities/Player.java [hunk 2]
```diff
@@ -23,6 +32,37 @@ public void setDefensePosition(boolean defense_position_) {
...
+	}
+	
+	public boolean win_xp(int xp_winned) {
```
Comment (LavergneC) [inline Classic_RPG/src/entities/Player.java]: Maybe not "win", "earn" sound better

Comment (LavergneC) [reply Classic_RPG/src/entities/Player.java]: winned -> won

**Review by LavergneC — CHANGES_REQUESTED** [review]
Minor changes before merge

File: Classic_RPG/src/entities/Player.java (hunk 1 above)
Comment (nicolasGibaud7) [reply Classic_RPG/src/entities/Player.java:45]: In a future pull request

File: Classic_RPG/src/game_management/Sections/Turn.java [hunk 3]
```diff
@@ -52,8 +52,15 @@ public boolean run_turn() {
 			
 
 			/* Remove monster from the wave if we get hp != 0 */
-			if (actual_wave.getMonsters().get(index_monstre).getHp() <= 0)
+			if (actual_wave.getMonsters().get(index_monstre).getHp() <= 0) {
+				boolean win_lvl = player.earn_xp(actual_wave.getMonsters().get(index_monstre).getXpGiven());
+				System.out.println("Xp : " + player.getXp());
 				actual_wave.removeMonster(index_monstre);
+				if (win_lvl) {
```
Comment (LavergneC) [inline Classic_RPG/src/game_management/Sections/Turn.java]: "win"

**Review by LavergneC — CHANGES_REQUESTED** [review]
A last win

**Review by LavergneC — CHANGES_REQUESTED** [review]
A last win

**Review by LavergneC — APPROVED** [review]

**Pull request merged**